libraries/Microsoft.Bot.Builder/TurnContext.cs

```diff
@@ -41,6 +41,37 @@ public TurnContext(BotAdapter adapter, Activity activity)
             Activity = activity ?? throw new ArgumentNullException(nameof(activity));
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TurnContext"/> class from another turncontext class.
```

Comment:
I would add something like "and sets the new instance activity property to the given activity" so the purpose of the constructor is more evident what it does from just reading the summary.